### PR TITLE
Linting fast-integration

### PR DESCRIPTION
**Pull request by lindnerby**
**Description**
There is a lot of JavaScript in the project that should as well be syntax checked, problem analysed and code style enforced. This PR introduces common linting tool: ESLint and fixes all lint issues.
For users of VSCode you may locally install the [ESLint extension](https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint) as well to make fixing issues easier.

Changes proposed in this pull request:

- Add ESLint to fast-integration package
- Configure Google-based linter rules (see https://github.com/google/eslint-config-google)
- Apply custom rules on top:
  - Exclude "require-jsdoc" rule, doesn't make sense in an integration test project
  - Extend "max-len" rule to 120, instead of just 80. We have a lot of long configuration strings and functions with many params. Shouldn't be a hassle to work with it
- Make changes to test/fast-integration/* to satisfy ESLint
- Create make targets for running linter

### CODE REVIEW

**Review by pbochynski — APPROVED** [review]

**Review by franpog859 — APPROVED** [review]
I love it

**Review by skhalash — DISMISSED** [review]

**Review by marcobebway — DISMISSED** [review]

File: tests/fast-integration/package-lock.json [hunk 1]
```diff
@@ -1,4391 +1,54 @@
 {
   "name": "fast-integration-tests",
   "version": "0.0.1-alpha.12",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
```
Comment (franpog859) [inline tests/fast-integration/package-lock.json]: Uh! One thing! :D I think we can stick to the second version of the lock file there

**Review by marcobebway — DISMISSED** [review]

**Review by skhalash — DISMISSED** [review]

**Review by jeremyharisch — APPROVED** [review]

**Review by marcobebway — APPROVED** [review]

**Review by skhalash — APPROVED** [review]

**Review by ralikio — DISMISSED** [review]

**Review by piotrmiskiewicz — DISMISSED** [review]

**Review by jeremyharisch — APPROVED** [review]

**Review by marcobebway — APPROVED** [review]

**Review by piotrmiskiewicz — APPROVED** [review]

**Review by skhalash — APPROVED** [review]

File: tests/fast-integration/utils/index.js [hunk 2]
```diff
@@ -378,335 +376,335 @@ async function k8sApply(resources, namespace, patch = true) {
...
+      `Waiting for Job ${name} to succeed ${success} timeout (${timeout} ms)`,
   );
 }
 
-async function kubectlExecInPod(pod, container, cmd, namespace = "default") {
-  let execCmd = [`exec`, pod, `-c`, container, `-n`, namespace, '--', ...cmd];
+async function kubectlExecInPod(pod, container, cmd, namespace = 'default') {
+  const execCmd = ['exec', pod, '-c', container, '-n', namespace, '--', ...cmd];
   try {
-    let out = await execa(`kubectl`, execCmd);
```
Comment (wozniakjan) [inline tests/fast-integration/utils/index.js:679]: this made the function pretty robust to failures, it's a try-catch block not executing anything..

**Pull request merged**